go/common/cbor/codec.go

```diff
@@ -10,7 +10,18 @@ import (
 )
 
 // Maximum message size.
-const maxMessageSize = 104857600 // 100 MiB
+const (
+	maxMessageSize = 104857600 // 100 MiB
+
+	// ModuleP2P corresponds to a peer-to-peer message context.
+	ModuleP2P = "p2p"
```

Comment:
These modules should not be defined in the codec package, they should be where the components use the codec.